### PR TITLE
[450] upgrade terraform to ver 0.13

### PR DESCRIPTION
**Pull request by felixtheflex**
### Context

upgrade terraform to ver 0.13 in order to leverage new features

### Changes proposed in this pull request

upgrade terraform to ver 0.13

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -91,8 +91,8 @@ jobs:
     - uses: actions/checkout@v2
     - name: Pin Terraform version
       uses: hashicorp/setup-terraform@v1
-      with:
-        terraform_version: 0.12.29
+      #with:
+      #  terraform_version: 0.12.29
```
Comment (vigneshmsft) [inline .github/workflows/build.yml:95]: can be pinned to a version `0.13.3` or `0.13.5`?

File: terraform/variables.tf [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (vigneshmsft) [inline terraform/variables.tf:1]: this blank line can be removed?

File: .github/workflows/build.yml (hunk 1 above)
Comment (felixtheflex) [reply .github/workflows/build.yml]: done...

**Review by vigneshmsft — APPROVED** [review]

**Review by dankmitchell — APPROVED** [review]

File: terraform/variables.tf (hunk 2 above)
Comment (felixtheflex) [reply terraform/variables.tf]: removed..

**Review by defong — APPROVED** [review]

**Pull request merged**